diff --git a/event-sauce/src/schema.rs b/event-sauce/src/schema.rs
index 4c3eb69..e3d045e 100644
--- a/event-sauce/src/schema.rs
+++ b/event-sauce/src/schema.rs
@@ -1,7 +1,9 @@
+#![allow(missing_docs)]
+
 table! {
     events (id) {
         id -> Uuid,
-        sequence_number -> Int4,
+        sequence_number -> Nullable<Int4>,
         event_type -> Varchar,
         entity_type -> Varchar,
         entity_id -> Uuid,
